src/twisted/python/_setup.py

```diff
@@ -121,23 +122,41 @@ def __init__(self, *args: object, **kwargs: object) -> None:
         Extension.__init__(self, *args, **kwargs)
 
 
+def _checkCPython(platform: Any = platform) -> bool:
+    """
+    Checks if this implementation is CPython.
+
+    This uses C{platform.python_implementation}.
+
+    This takes C{sys} and C{platform} kwargs that by default use the real
+    modules. You shouldn't care about these -- they are for testing purposes
+    only.
+
+    @return: C{False} if the implementation is definitely not CPython, C{True}
+        otherwise.
+    """
+    return cast(bool, platform.python_implementation() == "CPython")
+
+
+_isCPython = _checkCPython()  # type: bool
+
+
 # The C extensions used for Twisted.
-_EXTENSIONS = [
-    ConditionalExtension(
-        "twisted.test.raiser",
-        sources=["src/twisted/test/raiser.c"],
-        condition=lambda _: _isCPython,
-    ),
-    ConditionalExtension(
-        "twisted.internet.iocpreactor.iocpsupport",
-        sources=[
-            "src/twisted/internet/iocpreactor/iocpsupport/iocpsupport.c",
-            "src/twisted/internet/iocpreactor/iocpsupport/winsock_pointers.c",
-        ],
-        libraries=["ws2_32"],
-        condition=lambda _: _isCPython and sys.platform == "win32",
-    ),
-]
+_EXTENSIONS = (
+    [
+        ConditionalExtension(
+            "twisted.internet.iocpreactor.iocpsupport",
+            sources=[
+                "src/twisted/internet/iocpreactor/iocpsupport/iocpsupport.c",
+                "src/twisted/internet/iocpreactor/iocpsupport/winsock_pointers.c",
+            ],
+            libraries=["ws2_32"],
+            condition=lambda _: True,
+        ),
+    ]
+    if _isCPython and sys.platform == "win32"
```

Comment:
I think that this condition can be removed. The ConditionalExtension was introduced to handle this type of conditions :) and with that, I think the original _checkCPython can be kept at the end

Reply:
If this isn't present you get a tagged wheel

Reply:
Thanks. Then maybe it is worth adding a comment about why this extra condition is required :)

Reply:
I see the code from  _extension_kwargs ... well... _extension_kwargs might need a docstring